src/main/java/it/pagopa/pagopa/apiconfig/service/CdiService.java

```diff
@@ -152,6 +141,224 @@ public void deleteCdi(String idCdi, String pspCode) {
         cdiMasterRepository.delete(cdiMaster);
     }
 
+    public List<CheckItem> verifyCdi(MultipartFile file) {
```

Comment:
Quite long and complex method. Is it possible to divide it into sub-methods?

Reply:
👍 done where possible